configs/s2ef/200k/schnet/schnet_steps.yml

```diff
@@ -0,0 +1,27 @@
+includes:
+- configs/s2ef/200k/base.yml
+
+model:
+  name: schnet
+  hidden_channels: 1024
+  num_filters: 256
+  num_interactions: 3
+  num_gaussians: 200
+  cutoff: 6.0
+  use_pbc: True
+
+optim:
+  batch_size: 32
+  eval_batch_size: 16
+  num_workers: 64
+  lr_initial: 0.0005
+  lr_gamma: 0.1
+  lr_milestones: # epochs at which lr_initial <- lr_initial * lr_gamma
```

Comment:
i am looking at this now. Is this still epoch based or iteration based?

Reply:
Yeah, I'm supporting both. I was working on changing all the configs, but it became too tedious. So, I just added support for both now. Just set `update_lr_on_step` to True if you want to update by steps.